python/CMakeLists.txt

```diff
@@ -7,18 +7,7 @@ endif()
 
 
 if(USE_SYSTEM_PATHS_FOR_PYTHON_INSTALLATION)
-  if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
-    execute_process(
-      COMMAND "${PYTHON_EXECUTABLE}" -c "if True:
-  from distutils import sysconfig as sc
-  print(sc.get_python_lib(plat_specific=True))"
-      OUTPUT_VARIABLE Python3_SITEARCH
-      OUTPUT_STRIP_TRAILING_WHITESPACE)
-  else()
-    # Get install variable from Python3 module
-    # Python3_SITEARCH is available from 3.12 on, workaround if needed:
-    find_package(Python3 COMPONENTS Interpreter)
-  endif()
+  find_package(Python3 COMPONENTS Interpreter)
```

Comment:
I think we can leave the code as is. It's technically possible for someone to build this with cmake 3.10. But we know for sure our CI will have a newer version of cmake.